.eslintrc.js

```diff
@@ -91,11 +91,11 @@ module.exports = {
 				'no-restricted-globals': [
 					'error',
 					{
-						name: 'setInterval',
+						name: 'setTimeout',
 						message: 'Use the timers from @tldraw/util instead.',
 					},
```

Comment:
feel free to call it out of scope but this made me curious and then i had a hard time tracking down what it was actually referring to & how to use it. maybe tweak this message to say "Use the timers from `editor.timers`" instead?